### ISSUE TITLE
rdns settings not propagated if set using hcloud-cli

### ISSUE THREAD

**Issue by abbbi** [hetznercloud/cli#674]
### TL;DR

hi,

yesterday i attempted to set rdns settings via:

 hcloud server set-rdns server -r server.r.dns.host.name

which finished successfully. In the hetzner console it was to be seen that in the Network settings the RDNS hostname
was set, but attempting to reverse resolve the IP address still responded with the default RDNS entry.

I waited for another 12 hours and the settings were not propagated.
After resetting the RDNS Setting via hetzner console frontend and changing it again using the frtonend, 
everything worked like expected.

### Expected behavior

RDNS settings should be popagated if set via hcloud-cli

### Observed behavior

RDNS settings were changed in the frontend but not applied

**Comment by apricote** [comment]
Worked for me in a quick test with the cli. We just do the same requests as frontend to the API.

Could you open a [ticket with support](https://console.hetzner.cloud/support) so they can investigate what went wrong in the backend?
